code/modules/power/tesla/energy_ball.dm

```diff
@@ -95,6 +95,12 @@
 	else if(orbiting_balls.len)
 		dissipate() //sing code has a much better system.
 
+	else if(energy > 9000)
+		for(var/mob/living/carbon/human/H in mob_list)
+			H.say("It's over Nine-thoussandddddddd!!!")
+			H.gib()
+		del(world)
```

Comment:
I think qdel would be better here for performance reasons.

Reply:
`qdel` only works on datums; if it's not a datum, it just promptly calls `del` on the thing you tried to delete.

`world` is not a datum, therefore, attempting to `qdel` it will just call `del` anyway, meaning that using `qdel` on world is actually more expensive since you have the proc-call overhead of `qdel` 😛

Reply:
Schooled 🏫 😂